src/Entity/Beneficiaire.php

```diff
@@ -1036,4 +1036,13 @@ public function getAffiliatedRelays(): ReadableCollection
                 fn (BeneficiaireCentre $beneficiaryRelay) => $beneficiaryRelay->getCentre(),
             );
     }
+
+    public function hasValidSmsRelayInvitationCode(): bool
+    {
+        if (!$this->relayInvitationSmsCodeSendAt || !$this->relayInvitationSmsCode) {
```

Comment:
I would have used a ternary, and also maybee add a method $this->relayInvidationSmsSent()